packages/sourcecred/src/core/ledger/policies/balanced.js

```diff
@@ -26,6 +26,7 @@ export type Balanced = "BALANCED";
 export type BalancedPolicy = {|
   +policyType: Balanced,
   +budget: NonnegativeGrain,
+  numIntervalsLookback: number,
```

Comment:
plus sign should be used to make it readonly. We try to be strict in our typing.

Reply:
Fixed.